--- gcc/collect2.c.diff	2020-05-07 08:37:53.121608163 +0900
+++ gcc/collect2.c	2020-05-07 08:37:46.885903131 +0900
@@ -1534,7 +1534,7 @@
   if (redir)
     {
       /* Open response file.  */
-      redir_handle = open (redir, O_WRONLY | O_TRUNC | O_CREAT);
+      redir_handle = open (redir, O_WRONLY | O_TRUNC | O_CREAT, 0755);
 
       /* Duplicate the stdout and stderr file handles
 	 so they can be restored later.  */
